website/pages/en/querying/graphql-api.mdx

```diff
@@ -49,6 +49,25 @@ When querying a collection, the `orderBy` parameter may be used to sort by a spe
 }
 ```
 
+#### Example for nested entity sorting
+
+As of Graph Node [`v0.30.0`](https://github.com/graphprotocol/graph-node/releases/tag/v0.30.0) entities can be sorted on the basis of nested entities.
+
+In the following example, we sort the tokens by the name of their owner:
+
+```graphql
+{
+  tokens(orderBy: owner__name, orderDirection: asc) {
+    id
+    owner {
+      name
+    }
+  }
+}
+```
+
+> Currently you can sort by one-level deep `String` or `ID` types on `@entity` and `@derivedFrom` fields. Unfortunately, [sorting by interfaces on one level-deep entities](https://github.com/graphprotocol/graph-node/pull/4058) and sorting by nested entities is not supported yet.
```

Comment:
> sorting by nested entities is not supported yet
Do you mean arrays?

Reply:
both 
![CleanShot 2023-03-09 at 15 55 42](https://user-images.githubusercontent.com/44710980/224155786-c4ce1090-f9cd-4951-8da3-d92dadc34a43.png)

Reply:
I see what you mean - I think worth being very specific then, i.e. "sorting by fields which are arrays, or which themselves are nested entities is not supported". I wonder why we can't use the ID of the `AnotherEntity` to sort, as that will just be a string (except for derived fields)?

Reply:
> I wonder why we can't use the ID of the AnotherEntity to sort, as that will just be a string (except for derived fields)?

I think then it makes it go deeper in the tree. Technically it is just recursively going down the selection set. But we are limiting to 1 level cause of performance reasons.